janusgraph-core/src/main/java/org/janusgraph/diskstorage/keycolumnvalue/inmemory/ColumnValueStore.java

```diff
@@ -33,26 +35,31 @@
 
 /**
  * Implements a row in the in-memory implementation {@link InMemoryKeyColumnValueStore} which is comprised of
- * column-value pairs. This data is held in a sorted array for space and retrieval efficiency.
+ * column-value pairs. This data is held in a shared sorted array for space and retrieval efficiency, and is paged
+ * when/if data size exceeds a threshold to avoid excessive copying on updates.
  *
- * @author Matthias Broecheler (me@matthiasb.com)
  */
 
 class ColumnValueStore {
 
-    private static final double SIZE_THRESHOLD = 0.66;
+    public static final int DEF_PAGE_SIZE = 500;
```

Comment:
why public? if referred to in tests, make private or package private. private preferred so you can detect when the value is changed in source code here.

Reply:
done